raiden/api/python.py

```diff
@@ -642,15 +642,18 @@ def get_token_network_events(self, token_address, from_block, to_block):
                 'Expected binary address format for token in get_token_network_events'
             )
 
-        graph = self.raiden.token_to_channelgraph[token_address]
+        try:
+            graph = self.raiden.token_to_channelgraph[token_address]
 
-        return get_all_channel_manager_events(
-            self.raiden.chain,
-            graph.channelmanager_address,
-            events=ALL_EVENTS,
-            from_block=from_block,
-            to_block=to_block,
-        )
+            return get_all_channel_manager_events(
+                self.raiden.chain,
+                graph.channelmanager_address,
+                events=ALL_EVENTS,
+                from_block=from_block,
+                to_block=to_block,
+            )
+        except KeyError:
+            raise KeyError('The token address is not registered.')
```

Comment:
Instead of reraising a `KeyError` here, just use one of our own errors. I suggest using the [`UnknownTokenAddress`](https://github.com/raiden-network/raiden/blob/56d665056e03f8d67ea35a108ccb94086da6feaa/raiden/exceptions.py#L136) error.